utils/cautils/acme_flow.go

```diff
@@ -29,11 +29,14 @@ func ACMECreateCertFlow(ctx *cli.Context, provisionerName string) error {
 	}
 	ui.PrintSelected("Certificate", certFile)
 
-	_, err = pemutil.Serialize(af.priv, pemutil.ToFile(keyFile, 0600))
-	if err != nil {
-		return errors.WithStack(err)
+	// We won't have a private key with attestation certificates
+	if af.priv != nil {
+		_, err = pemutil.Serialize(af.priv, pemutil.ToFile(keyFile, 0600))
+		if err != nil {
+			return errors.WithStack(err)
+		}
+		ui.PrintSelected("Private Key", keyFile)
```

Comment:
It may be an option to indicate that the private key is created and stored in the KMS for the attestation flow?

Reply:
Fixed with 9a45100